hosts/signed-int16/home/reisen/home.nix

```diff
@@ -30,19 +30,23 @@
       kdenlive
 
       # Media
-      quodlibet
+      quodlibet-full
       vlc
       yt-dlp
       brasero
-      cdrtools
+      ardour
+      natron
 
       # Dev
       lazygit
+      jetbrains.pycharm-community
+      android-studio
 
       # Gaming
       steam
       steam-tui
       steam-run
```

Comment:
no, never do stuff like that

Reply:
i don't even remember when or why i did this, i'll get rid of it